packages/styles/scss/components/carousel/_carousel.scss

```diff
@@ -89,9 +89,13 @@
   }
 
   .#{$prefix}--btn.#{$prefix}--carousel__navigation__btn {
-    padding: calc((#{$container-02} - #{$icon-size-01}) / 2);
+    padding: 0;
     min-height: $container-02;
     width: $container-02;
```

Comment:
@kennylam is there a container-03 that would equal 40px? The buttons are rendering at 32 x 32px now

Reply:
@oliviaflory Good catch, I forgot to bump up the size after removing the padding.